pkgs/tools/security/aeskeyfind/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aeskeyfind";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://citpsite.s3.amazonaws.com/memory-content/src/aeskeyfind-${version}.tar.gz";
+    sha256 = "sha256-FBflwbYehruVJ9sfW+4ZlaDuqCR12zy8iA4Ev3Bgg+Q=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp aeskeyfind $out/bin
+    runHook postInstall
+  '';
```

Comment:
Why is the program not being compiled but instead an already compiled binary is used?

Reply:
I believe it is. I will look into this more latter today.

Reply:
You're right, the binary is not in the source. So somehow it is being compiled. I guess that using to the Makefile is the default of the buildPhase. In conclusion this is solved.

Reply:
`stdenv` provides `gcc` and `make`.